Assets/Mirror/Editor/Weaver/Extensions.cs

```diff
@@ -137,5 +137,15 @@ public static MethodReference MakeHostInstanceGeneric(this MethodReference self,
             return Weaver.CurrentAssembly.MainModule.ImportReference(reference);
         }
 
+        public static CustomAttribute GetCustomAttribute(this MethodDefinition method, string attributeName)
```

Comment:
why is this still needed? doesn't it work without [NetworkReader] attributes?

Reply:
This is not used to find NetworkReader.   It is used to find out if a method is an extension method.  

To find out if a method is an extension method,  I need to check if it has `System.Runtime.CompilerServices.ExtensionAttribute` custom attribute.   This method is to find that attribute.

Reply:
I see